K2Bridge/Controllers/IndexListController.cs

```diff
@@ -13,6 +13,8 @@ namespace K2Bridge.Controllers
     /// Handles Index List requests.
     /// The original request produced by Kibana is in the format of:
     /// POST /*/_search?ignore_unavailable=true HTTP/1.1.
+    /// In addition, for Kibana 7:
```

Comment:
I think 'old' api calls should be removed from the code, meaning lets not try to aim for both Kibana 6.8 and 7.

We can either removed those calls in this PR.
or open a new task to removed redundant calls.
wdyt

Reply:
Completely agree, however in this case, Kibana 7 *also* sends a `/_search` request, so both `_resolve` and `_search` seem necessary.